crates/rerun/src/run.rs

```diff
@@ -98,6 +98,10 @@ struct Args {
     #[clap(long)]
     screenshot_to: Option<std::path::PathBuf>,
 
+    /// Do not display the welcome screen.
+    #[clap(long)]
+    skip_welcome_screen: bool,
```

Comment:
This is never used?

Reply:
Good question. I (prematurely?) added it thinking that it would be needed to avoid the welcome screen temporarily showing up in this situation:
```
rr.init("delay", spawn=True)
time.sleep(5)
rr.log_points("points", np.random.rand(10, 2))
```
However, `rr.init()` itself sends stuff (at least the app ID, etc.) which gets us directly into the "active blueprint and recording" state. So for _this_ scenario, this `--skip-welcome-screen` is not needed. I assume it will be the same for the C++ SDK. Rust uses the `Sdk` channel, which skips the welcome screen until the recording is closed, so that's ok.

Beyond the web demo, I haven't tested all other scenarios yet though—I wouldn't even be capable to list them tbh. But I don't really expect we have an immediate need for that flag yet. I _can_ think of future scenarios where this could come handy(e.g. long running server where advanced users don't want to see "that annoying screen"), but I'm totally fine removing it for the time being.

 👍🏻👎🏻❓

Reply:
Never mind the above 🦆 

The Python SDKs really want that `--skip-welcome_screen`. I can easily see it by adding delays in the `spawn()` function. So it means that on a slow computer, or due to future change, that welcome screen could temporarily "flash" before data is displayed, which is far worse than the loading screen. So I'll use that flag in the python SDK (and C++ if I can figure out how).